src/boot/efi/stdclib.c

```diff
@@ -0,0 +1,74 @@
+/* Public domain */
+/* Taken from https://clc-wiki.net/wiki */
+#include <stdint.h>
+
+#include "stdclib.h"
+
+/* These are C library functions required by libfdt */
+
+void *memmove(void *dest, const void *src, size_t n) {
```

Comment:
I don't like that this is necessary, but I guess it's isolated enough. I am wondering though, can these be "implemented" by just calling __builtin_memmove etc.?